tensorflow_estimator/python/estimator/early_stopping.py

```diff
@@ -177,13 +181,18 @@ def read_eval_metrics(eval_dir):
 
 def _stop_if_threshold_crossed_hook(estimator, metric_name, threshold,
                                     higher_is_better, eval_dir, min_steps,
-                                    run_every_secs, run_every_steps):
+                                    min_delta, run_every_secs, run_every_steps):
   """Creates early-stopping hook to stop training if threshold is crossed."""
 
   if eval_dir is None:
     eval_dir = estimator.eval_dir()
 
-  is_lhs_better = operator.gt if higher_is_better else operator.lt
+  def is_lhs_better(lhs_val, val):
```

Comment:
I would rename the function now that it's not "only" better, but better by min_delta